### PR TITLE
DEVX-1202: fix docs build

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -792,12 +792,13 @@ solution, Confluent Replicator is also configured with security.
 
 9. Restart the Replicator connector.
 
-10. Observe that the ``connect-replicator`` consumer group has resumed
-   consumption. Notice several things:
+10. Observe that the ``connect-replicator`` consumer group has resumed consumption. Notice several things:
```
Comment (tlblessing) [inline docs/index.rst]: Might want to use #. instead of hard-coding numbers. It might auto-accommodate your spacing alignment.

Comment (ybyzek) [reply docs/index.rst]: @tlblessing Yes, 💯 .  But this whole docs page needs to be converted.  Can we approve/ this PR to quickly fix the build, and then in a separate PR address the `#.` ?

Comment (ybyzek) [reply docs/index.rst]: https://confluentinc.atlassian.net/browse/DEVX-1378

Comment (tlblessing) [reply docs/index.rst]: sure.

**Comment by ybyzek** [comment]
Docs build PR passes, merging now

**Pull request merged**